index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>N & K cakes</title>
+    <link rel="stylesheet" href="style.css">
+    <script  src="js/index.js"></script>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <div class="page-wrapper">
+    <!--HEADER-->
+<div id="home">
+        <div class="header" id="header">
+
+        <!--NAV-->
+        <div id="header-img">
+          <img src="images\logo.png" alt="">
+        </div>
+
+
+            <nav id="nav-bar">
+              <a href="#home" class="nav-link">Home</a>
+              <a href="#about" class="nav-link">About Us</a>
+              <a href="#contact" class="nav-link">Contact</a>
+            </nav>
```

Comment:
Let's shift the navigation section to a semantic tag `<header> `accordingly to the best practice.